tests/conftest.py

```diff
@@ -592,3 +592,10 @@ def patch_bypass_setup(model: str):
     klass = "".join([w.capitalize() for w in model.split("_")] + ["Model"])
     object_to_patch = f"virtual_ecosystem.models.{model}.{model}_model.{klass}"
     return patch(f"{object_to_patch}._bypass_setup_due_to_static_configuration")
+
+
+def patch_static_config(model: str):
+    """Patch the run_update_check udring the init of the model."""
+    klass = "".join([w.capitalize() for w in model.split("_")] + ["Model"])
+    object_to_patch = f"virtual_ecosystem.models.{model}.{model}_model.{klass}"
+    return patch(f"{object_to_patch}._check_static_config")
```

Comment:
I think it would be easier to work with the model object here and in the other patches above:
```suggestion
def patch_static_config(model: type):
    """Patch the run_update_check udring the init of the model."""
    return patch(f"{model.__module__}.{model.__name__}._check_static_config")
```
```

Reply:
Much cleaner, indeed. To be honest, it did not occurred to me. I'll make the changes.

Reply:
Sorted in [e7689cd](https://github.com/ImperialCollegeLondon/virtual_ecosystem/pull/693/commits/e7689cdd8ad2b30f231a6d7a4182193a98e4bd64)